tweepy/pagination.py

```diff
@@ -61,9 +64,9 @@ def flatten(self, limit=inf):
 
 
 class PaginationIterator:
-
```

Comment:
Removing this newline is inconsistent with elsewhere in the library.
In the future, please refrain from including unnecessary formatting changes in pull requests.